### PR TITLE
update resource file format list

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

File: ckanext/canada/schemas/presets.yaml [hunk 1]
```diff
@@ -3166,6 +3178,10 @@ presets:
     - value: RDF
       mimetype: application/rdf+xml
       openness_score: 4
+    - value: RDF/XML
+      mimetype: application/rdf+xml
```
Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]: looks like a duplicate of RDF

File: ckanext/canada/schemas/presets.yaml [hunk 2]
```diff
@@ -3089,13 +3091,18 @@ presets:
     - value: GeoTIF
       mimetype: image/tiff
       replaces: [geotif]
+    - value: GeoJSON
+      mimetype: application/geo+json
+    - value: GeoPackage
```
Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]: I've tried to use the accepted file extensions as values in this list (e.g. `GPKG` for GeoPackage) and use the names of the formats as labels so they can be translated.

File: ckanext/canada/schemas/presets.yaml [hunk 3]
```diff
@@ -3166,6 +3178,10 @@ presets:
     - value: RDF
       mimetype: application/rdf+xml
       openness_score: 4
+    - value: RDF/XML
+      mimetype: application/rdf+xml 
+    - value: RDF Turtle
```
Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]: Value should be `TTL`, use `label: RDF Turtle`

**Comment by wardi** [comment]
Using all-uppercase versions of file extensions is a pattern we're inheriting from CKAN. We should avoid storing format names (use labels instead).

As a future improvement now that we're on a more recent version of CKAN we can also use CKAN's built in mapping of file extensions to mime-types (extending it as necessary) instead of having them embedded here.

File: ckanext/canada/schemas/presets.yaml (hunk 1 above)
Comment (fanjinfei) [inline ckanext/canada/schemas/presets.yaml]: since XML is also an existing entry, so is this RDF/XML still needed at all?

Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]: IIUC the RDF format above is the same as the RDF/XML you're adding here. XML is a more generic type

**Comment by fanjinfei** [comment]
possible mapinfo file extension: *.TAB, *.MAP, *.DAT, *.ID,  *.IND.  Shall we include them all?
there is no specific extension for sql lite, could be .db, .db3, .SQLite, .sqlite3.  How to handle this?

**Comment by wardi** [comment]
For MapInfo I'd choose `TAB` based on the description in http://www.gdal.org/drv_mitab.html

`SQLITE3` is good.

File: ckanext/canada/schemas/presets.yaml [hunk 4]
```diff
@@ -3186,17 +3204,25 @@ presets:
...
+      value: SQLITE
     - value: SVG
       mimetype: image/svg+xml
       replaces: [svg]
+    - value: TAB
     - value: TIFF
       mimetype: image/tiff
       replaces: [tiff]
+    - value: TRiG
+    - value: TRiX
```
Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]:
```
TRIG
TRIX
```

File: ckanext/canada/schemas/presets.yaml [hunk 5]
```diff
@@ -3186,17 +3204,25 @@ presets:
...
       mimetype: application/sql
       replaces: [sql]
+    - label: SQL Lite3
+      value: SQLITE3
+    - label: SQL Lite
+      value: SQLITE
     - value: SVG
       mimetype: image/svg+xml
       replaces: [svg]
+    - value: TAB
```
Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]: repeated from above

File: ckanext/canada/schemas/presets.yaml [hunk 6]
```diff
@@ -3117,6 +3125,7 @@ presets:
     - value: JSON
       mimetype: application/json
       openness_score: 3
+    - value: JSON-LD
```
Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]: looks like the file extension has no hyphen. so `JSONLD`, optionally with a label.

**Pull request merged**